main/src/main/scala/sbt/Keys.scala

```diff
@@ -225,6 +225,12 @@ object Keys {
   val scalaArtifacts = settingKey[Seq[String]]("Configures the list of artifacts which should match the Scala binary version").withRank(CSetting)
   val enableBinaryCompileAnalysis = settingKey[Boolean]("Writes the analysis file in binary format")
   val crossJavaVersions = settingKey[Seq[String]]("The java versions used during JDK cross testing").withRank(BPlusSetting)
+  val enableSemanticDB = settingKey[Boolean]("Enables SemanticDB Scalac plugin").withRank(CSetting)
```

Comment:
For consistency with sbt-scoverage, I propose `semanticdbEnabled := true`, https://github.com/scoverage/sbt-scoverage#how-to-use